networks/cardano/tokens/cc8d1b026353022abbfcc2e1e71159f9e308d9c6e905ac1db24c7fb6.json

```
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "paribus",
+    "decimals": 6,
+    "mldId": "cardano-PBX-cc8d1b026353022abbfcc2e1e71159f9e308d9c6e905ac1db24c7fb6",
+    "orgName": "Paribus",
+    "orgUrl": "https://paribus.io/",
+    "tokenAddress": "cc8d1b026353022abbfcc2e1e71159f9e308d9c6e905ac1db24c7fb6",
```

```suggestion
    "tokenAddress": "cc8d1b026353022abbfcc2e1e71159f9e308d9c6e905ac1db24c7fb650617269627573",
```
the asset name is missing, the same goes for `mldId` and this file name

---

@rebelC0der please update the file name and `mldId` too